frontend/src/components/ViewTrips/edit-trip-button.js

```diff
@@ -25,7 +21,7 @@ const EditTripButton = (props) => {
     <Button
       type='button'
       variant='link'
-      onClick={() => props.handleEditTrip(props.tripId, props.formattedTripData)}
+      onClick={() => props.handleEditTrip(props.tripId, props.tripData)}
```

Comment:
```suggestion
      onClick={props.handleEditTrip(props.tripId, props.tripData)}
```

I believe this works, you'll have to test it out though.

Reply:
This is what I tried originally and it actually causes some problems if you try to pass in parameters. This way works because technically it is creating a new function each time onClick. See [here](https://stackoverflow.com/questions/29810914/react-js-onclick-cant-pass-value-to-method) for more info.